doc/lua_api.txt

```diff
@@ -3345,6 +3345,19 @@ It can be created via `Settings(filename)`.
     * write changes to file
 * `to_table()`: returns `{[key1]=value1,...}`
 
+### `RaycastState`
+A raycast on the map. It works with selection boxes.
+
+It can be created via `RaycastState(pos1, pos2, objects, liquids)` where:
```

Comment:
I think `Raycast` would be a better name, since obvious that these object-style functions/tables will have some state.